src/collections/_documentation/performance/alerts-notifications/notifications.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Other Notifications
+sidebar_order: 2
+---
+
+Aside from Alerts, Sentry has other types of notifications, such as Workflow, Deploy, and Quota notifications. All these notifications are sent via email.
```

Comment:
Remove via here, perhaps "using" or "to your" or "to the defined email for the project"